ingestion/src/metadata/ingestion/source/database/exasol/queries.py

```diff
@@ -0,0 +1,22 @@
+import textwrap
+
+# TODO verify sql text would contain dbt & openmetadata like that
+EXASOL_SQL_STATEMENT = textwrap.dedent(
```

Comment:
- [ ] Check that EXASOL_SQL_STATEMENT + filters are sufficient for extracting lineage